zanata-war/pom.xml

```diff
@@ -163,6 +161,7 @@
         <groupId>com.ning.maven.plugins</groupId>
         <artifactId>maven-duplicate-finder-plugin</artifactId>
         <configuration>
+          <checkTestClasspath>false</checkTestClasspath>
```

Comment:
Do we have two copies of API interfaces (in which case, we should fix it by blocking one of the transitive dependencies), or do we actually have two different versions of implementation code?  If it's the latter, perhaps you could move something out of zanata-war to make it work.

In any case, `<checkTestClasspath>false</checkTestClasspath>` is not something we should do. :-1: